projects/angular-sdk/src/components/global.component.ts

```diff
@@ -58,6 +61,10 @@ export class GlobalComponent implements OnChanges {
     // Life Cycle Hooks
     // -----------------------------------------------
 
+    ngOnInit(): void {
```

Comment:
If you add a **ngOnInit** method (angular life cycle) on the **global** component, you need to add a call to this method into the NgOnInit methods of child components (if exists), otherwise the **ngOnInit** methods in child components will override the global "ngOnInit".

You can do this : 

```
// Child components
ngOnInit(): void {
    // Parent
    super.ngOnInit();
    
    // existing code
    .....
}
```

Reply:
oh good to know! updated

Reply:
let me know if i missed any places